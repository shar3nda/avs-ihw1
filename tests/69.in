10
10 3 -1 4 -1 10 6 4 -1 8