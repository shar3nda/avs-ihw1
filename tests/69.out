-1 4 -1 10 6 4 -1 8